lib/pakyow/app/assets/console/styles/console.scss

```diff
@@ -334,6 +340,38 @@ aside {
     color: darken($color-background, 25%);
     margin-top: 0;
   }
+
+  @include mobile {
+    width: 100%;
+    padding: 5px 0;
+    height: 25px;
+    top: 0;
+    right: 0;
+    margin-left: 0 !important;
+    display: inline;
+    left: 0% !important;
+    margin-top: -5px;
+
+    nav {
+      background-color: $color-xgray;
+      padding: 10px 0;
+      margin-bottom: 10px !important;
+      border-bottom: 1px solid $color-border;
+      overflow-x: auto;
+    }
+
+    ul {
+      width: 90%;
+      margin: auto;
+      text-align: center;
+      overflow-x: auto;
+    }
+
+    li {
+      display: inline !important;
+      margin-left: 5.5%;
```

Comment:
I wouldn't worry about font size changes between the two. So long as we aren't changing the font size values ourselves we can trust that the OS is doing the right thing to scale the UI for whatever the display resolution is. IMO we over-complicate it by trying to compensate.